ground/src/main/java/com/google/android/ground/util/StringExt.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.util
+
+fun String?.isNotNullOrEmpty(): Boolean = !this.isNullOrEmpty()
```

Comment:
Sorry I thought this existed already. If not I don't know if it's worth adding as an ext function, your call.

Reply:
Let's keep it for now.